app/ts/background/simulationModeHanders.ts

```diff
@@ -98,6 +99,36 @@ export async function sendTransaction(
 	)
 }
 
+export async function sendRawTransaction(
+	websiteTabConnections: WebsiteTabConnections,
+	getActiveAddressForDomain: (websiteAccess: WebsiteAccessArray, websiteOrigin: string, settings: Settings) => bigint | undefined,
+	ethereumClientService: EthereumClientService,
+	sendRawTransactionParams: SendRawTransaction,
+	socket: WebsiteSocket,
+	request: InterceptedRequest,
+	simulationMode: boolean,
+	website: Website,
+	settings: Settings
+) {
+	const transaction = ethers.Transaction.from(dataStringWith0xStart(sendRawTransactionParams.params[0]))
+	
+	const from = getFromField(websiteTabConnections, simulationMode, transaction.from === null ? undefined : EthereumAddress.parse(transaction.from), getActiveAddressForDomain, socket, settings)
+	const sendTransactionParams: SendTransactionParams = {
+		method: 'eth_sendTransaction',
+		params: [{
+			from: from,
+			data: stringToUint8Array(transaction.data),
+			...transaction.gasLimit === null ? { gas: transaction.gasLimit } : {},
+			value: transaction.value,
+			...transaction.to === null ? {} : { to: EthereumAddress.parse(transaction.to) },
+			...transaction.gasPrice === null ? {} : { gasPrice: transaction.gasPrice },
+			...transaction.maxPriorityFeePerGas === null ? {} : { maxPriorityFeePerGas: transaction.maxPriorityFeePerGas },
+			...transaction.maxFeePerGas === null ? {} : { maxFeePerGas: transaction.maxFeePerGas },
+		}]
+	}
+	return sendTransaction(websiteTabConnections, getActiveAddressForDomain, ethereumClientService, sendTransactionParams, socket, request, simulationMode, website, settings)
```

Comment:
I'm not sure this is what we want to do.  If the dapp gives us a raw transaction, we shouldn't be calling `eth_sendTransaction` on the connected wallet, we should be calling either `eth_sendRawTransaction` against our RPC or we should be calling `eth_sendRawTransaction` on MetaMask (whichever is appropriate).

In this situation, we don't actually need a connected wallet, so we could do this even in simulation mode.

Reply:
We are converting `eth_sendRawTransaction` to `eth_sendTransaction` here. Is there a reason why we wouldn't want to be doing that?

Reply:
It is very possible (and arguably to-spec) that wallets will ignore any signature properties included in an `eth_sendTransaction` call, which would result in the user getting prompted to sign the transaction in their wallet.  The transaction being submitted is likely not `from` the wallet account, so this likely would result in stuff breaking if we implement it this way.

Reply:
yeah, I am prompting user to sign the transaction here. Right, that's incorrect. I feel we should still present the transaction to user and the user will then decide to forward or not forward it

Reply:
I think it is reasonable to show the transaction to the user before broadcasting it, we just need to broadcast it as a signed transaction (`eth_sendRawTransaction`) not as an unsigned transaction (`eth_sendTransaction`).